### PR TITLE
react_tabs-js-state

### PR DESCRIPTION
**Pull request by vintwp**
https://vintwp.github.io/react_tabs-js/

### CODE REVIEW

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]
* test

File: src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  const [selectedTabId, setSelectedTabId] = useState('tab-1');
+
+  return (
+    <div className="section">
+      <h1 className="title">
+        Selected tab is
+        {' '}
+        {
+          tabs.find(tab => tab.id === selectedTabId).title
+        }
```
Comment (DarkMistyRoom) [inline src/App.jsx:24]: better use interpolation

File: src/components/Tabs/Tabs.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                'is-active': correctTabId === tab.id,
+              })}
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={(event) => {
+                  if (tab.id !== correctTabId) {
+                    onTabSelected(tab);
+                  }
```
Comment (DarkMistyRoom) [inline src/components/Tabs/Tabs.jsx:31]: move this logic to a handler function

**Review by DarkMistyRoom — APPROVED** [review]
Good job! Check the comments

**Pull request closed without merging**